crates/forge/src/coverage.rs

```diff
@@ -140,8 +142,12 @@ impl CoverageReporter for LcovReporter {
                             writeln!(out, "FNDA:{hits},{name}")?;
                         }
                     }
-                    CoverageItemKind::Line => {
-                        writeln!(out, "DA:{line},{hits}")?;
+                    // Add lines / statement hits only once.
+                    CoverageItemKind::Line | CoverageItemKind::Statement => {
+                        if !recorded_lines.contains(&line) {
```

Comment:
if insert() directly

Reply:
yep, simplified in https://github.com/foundry-rs/foundry/pull/10423/commits/7c58c573813af991b3001c832d3250481966dd71